lib/galaxy/datatypes/sequence.py

```diff
@@ -1206,25 +1206,24 @@ def sniff_prefix(self, file_prefix: FilePrefix) -> bool:
         >>> Axt().sniff( fname )
         False
         """
-        headers = get_headers(file_prefix, None)
-        if len(headers) < 4:
+        headers = get_headers(file_prefix, None, count=4, comment_designator="#")
+        if not (
+            len(headers) == 4
+            and len(headers[0]) == 9
```

Comment:
Could we instead do this ?
```suggestion
            and headers[0] == "matrix=axt"
```
that seems even more specific

Reply:
It would, but it is not a part of the standard as defined by https://genome.ucsc.edu/goldenPath/help/axt.html

Some tools add that comment line, others don't

Reply:
Doesn't that mean you can't make that check at all, since now you're relying on this line having 9 characters, which feels fairly random ?

Reply:
Ah, this is a non-comment line, nevermind.